### PR TITLE
fix pip install command

### PR DESCRIPTION
**Pull request by stefanhoelzl**

### CODE REVIEW

**Comment by MatthieuDartiailh** [comment]
Thanks @stefanhoelzl ! Stupid copy-paste mistake. Happy to know that people actually read the docs.

File: docs/source/basis/installation.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ Wheels
 If you don't use Anaconda, you can install Kiwisolver pre-compiled,
 through PIP, for most common platforms::
```
Comment (Qix-) [inline docs/source/basis/installation.rst]: Double colon by the way ;)

**Comment by MatthieuDartiailh** [comment]
The double colon is rst to indicate that the next paragraph is a literal block http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks.

**Comment by Qix-** [comment]
Ahh TIL, disregard :)

**Pull request merged**